pkg/ddevapp/config.go

```diff
@@ -420,3 +453,60 @@ func prepLocalSiteDirs(base string) error {
 
 	return nil
 }
+
+// validateCommandYaml validates command hooks and tasks defined in extend-commands for config.yaml
```

Comment:
I had a similar issue. When I used the example from before:

```
extend-commands:
  post-import-db:
    - exec2: "drush cc all"
```

Note the issue with `exec2` being an invalid ddev command. I then get the following panic:

```
➜  drupal-kickstart git:(master) ✗ ddev import-db --src ~/Downloads/db.tar.gz --extract-path kickstart.sql
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x13bd168]

goroutine 1 [running]:
github.com/drud/ddev/pkg/plugins/platform.(*LocalApp).GetName(0xc420086030, 0x0, 0x18999c0)
	/go/src/github.com/drud/ddev/pkg/plugins/platform/local.go:151 +0x8
github.com/drud/ddev/cmd/ddev/cmd.glob..func7(0x18cebe0, 0xc42007da80, 0x0, 0x4)
	/go/src/github.com/drud/ddev/cmd/ddev/cmd/import-db.go:43 +0x317
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).execute(0x18cebe0, 0xc42007da40, 0x4, 0x4, 0x18cebe0, 0xc42007da40)
	/go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:603 +0x22b
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x18cf8a0, 0x0, 0x18cf8a0, 0xc42012df50)
	/go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:689 +0x339
github.com/drud/ddev/vendor/github.com/spf13/cobra.(*Command).Execute(0x18cf8a0, 0xc42012df68, 0x14b9b41)
	/go/src/github.com/drud/ddev/vendor/github.com/spf13/cobra/command.go:648 +0x2b
github.com/drud/ddev/cmd/ddev/cmd.Execute()
	/go/src/github.com/drud/ddev/cmd/ddev/cmd/root.go:98 +0x3b
main.main()
	/go/src/github.com/drud/ddev/cmd/ddev/main.go:6 +0x20
```